src/executePrettier.js

```diff
@@ -37,7 +47,15 @@ const executePrettier = (editor, text) => {
           filePath: getCurrentFilePath(editor),
         }));
     }
-    return prettier.format(text, getPrettierOptions(editor));
+
+    const prettierOptions = getPrettierOptions(editor);
+    const localPrettier = getLocalPrettierPath(getCurrentFilePath(editor));
+
+    if (!localPrettier) {
+      return prettier.format(text, prettierOptions);
+    }
+
+    return getLocalPrettier(localPrettier).format(text, prettierOptions);
```

Comment:
I don't know if I'd do the `format` thing, then you're putting in the logic in two different places that otherwise had no need to know anything about how `executePrettier` works?